src/main/java/bio/terra/service/tabulardata/google/BigQueryProject.java

```diff
@@ -41,7 +43,14 @@ public final class BigQueryProject {
   private BigQueryProject(String projectId) {
     logger.info("Retrieving Bigquery project for project id: {}", projectId);
     this.projectId = projectId;
-    bigQuery = BigQueryOptions.newBuilder().setProjectId(projectId).build().getService();
+    HttpTransportOptions transportOptions = StorageOptions.getDefaultHttpTransportOptions();
+    transportOptions =
+        transportOptions.toBuilder()
+            .setConnectTimeout(40 * 1000)
+            .setReadTimeout(40 * 1000)
```

Comment:
What are these magic numbers?

Reply:
These are the values currently set in the gcs config/app properties for these values. I might be struggling a bit trying to figure out how to wire them in.

Reply:
I attempted a few approaches to map this in from a configuration, but it's not going smoothly. I'd like to get it merged just with a constant. 

I set them to 40 b/c that is what we currently have set in the gcsConfiguration for the same settings.